scattering/utils/features.py

```diff
@@ -59,6 +59,10 @@ def find_all_minima(arr):
 
     checks = np.r_[True, arr[1:] < arr[:-1]] & np.r_[arr[:-1] < arr[1:], True]
     minima = np.where(checks)[0]
+    global_min = min(arr)
+    global_r = np.where(arr == global_min)[0]
+    minima = np.append(minima, global_r)
+    minima.sort()
```

Comment:
If the global minima is also a local minima, would this not create a duplicate entry in the result array? Other than that, this look good

Reply:
@lisankim0321 could you double-check this for the x^2 function?

The reason for adding this is that the `checks` variable on line 60 doesn't correctly capture cases when a curve starts at the global minima and continues to increase as x increases.